### PR TITLE
C fasta

### PR DESCRIPTION
**Pull request by vext01**
Here is the C fasta with checksums.

I have another branch that begins adding the checksums to other languages:
https://github.com/softdevteam/warmup_experiment/tree/fasta_checksums

I will come back to that later.

As discussed with CF, due to time constraints we are going to do all the C benchmarks first.

OK?

### CODE REVIEW

File: benchmarks/fasta/c/bench.c [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+/*
+ * Used to generate a checksum to verify benchmark correctness
+ */
+void wrap_write(int fd, char *buf, size_t len) {
+   int i;
+
+   for (i = 0; i < len; i++) {
+      /* In languages like Python, we would modulo 2^{32} */
```
Comment (cfbolz) [inline benchmarks/fasta/c/bench.c]: I don't think we need this comment, do we?

Comment (vext01) [inline benchmarks/fasta/c/bench.c]: Not really. It was to demonstrate the idea. Will remove.

**Comment by vext01** [comment]
OK to squash?

**Comment by cfbolz** [comment]
looks good, merging

**Pull request merged**